packages/bpk-tokens/src/ios/base/aliases.json

```diff
@@ -28,6 +28,18 @@
     "SPACING_XXL": 40,
     "BORDER_SIZE_SM": 1,
     "BORDER_SIZE_LG": 2,
-    "BORDER_SIZE_XL": 3
+    "BORDER_SIZE_XL": 3,
+    "SHADOW_OFFSET_HEIGHT_SM": 1,
```

Comment:
Hmm, so you have opted for {TOKEN_NAME}_{PROPERTY_NAME}_{MODIFIER}`

Interestingly we use `{TOKEN_NAME}_{MODIFIER}_{PROPERTY_NAME}` elsewhere (see text tokens https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/tokens/ios/base.raw.json#L1028).

Reply:
We use `{TOKEN_NAME}_{MODIFIER}_{PROPERTY_NAME}` with buttons too, see https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/src/web/base/buttons.json

Reply:
I'll update to be consistent with the rest of the tokens!